frontend/src/lib/components/Cards/InsightCard/InsightCard.tsx

```diff
@@ -561,10 +561,11 @@ function InsightCardInternal(
     const exportedAndCached = dashboardPlacement && dashboardPlacement == DashboardPlacement.Export && !!insight.result
     const sharedAndCached = dashboardPlacement && dashboardPlacement == DashboardPlacement.Public && !!insight.result
     const canMakeQueryAPICalls =
-        dashboardPlacement &&
-        [DashboardPlacement.Dashboard, DashboardPlacement.ProjectHomepage, DashboardPlacement.FeatureFlag].includes(
-            dashboardPlacement
-        )
+        (dashboardPlacement &&
+            [DashboardPlacement.Dashboard, DashboardPlacement.ProjectHomepage, DashboardPlacement.FeatureFlag].includes(
+                dashboardPlacement
+            )) ||
+        dashboardPlacement === undefined // saved insights grid
```

Comment:
Why not `DashboardPlacement.SavedInsights` ?

Reply:
Hmmm, "SavedInsights" isn't a dashboard. 

Have changed the name so that it is the card's placement. Which sometimes is _also_ a DashboardPlacement. And other times is "SavedInsightsGrid"

Reply:
other than in storybook, those are the only (current) valid placements for an insight card